scripts/changelog-csv.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+readonly CSV_FILE='commits.csv'
+# Temporary file for new entries
+NEW_ENTRIES=$(mktemp)
+# Backup file for existing CHANGELOG
+CHANGELOG_BACKUP="CHANGELOG.md.bak"
```

Comment:
```suggestion
CHANGELOG_BACKUP="CHANGELOG.md.bak-$(date +%s)"
```
this should make it unique that you don't wipe your backup with your backup (alternatively: is backing it up actually useful assuming git? there is also the `temp_changelog` so seems like there are actually three versions of it during the lifetime of the script where two could be sufficient?)